wallet-core/ios-guide.md

```diff
@@ -0,0 +1,3 @@
+# Sample iOS Wallet -- Step-by-step Guide
+
```

Comment:
can we add this file later?

Reply:
This is early WIP, I created PR so others can see it.

Reply:
for WIP use draft mode for PR's